main.go

```diff
@@ -43,6 +44,10 @@ var (
 	dump      = app.Command("dump", "Display information about a certificate.")
 	dumpFiles = dump.Arg("file", "Certificate file to dump (or stdin if not specified).").ExistingFiles()
 	dumpType  = dump.Flag("format", "Format of given input (heuristic guess if not specified).").String()
+
+	connect     = app.Command("connect", "Connect to a server and print its certificate")
+	connectTo   = connect.Arg("server:port", "Hostname or IP to connect to").String()
+	connectName = connect.Flag("name", "Override the server name used for SNI").String()
```

Comment:
http://www.nytimes.com/2016/06/10/world/europe/period-full-stop-point-whatever-its-called-millennials-arent-using-it.html